Lib/blackrenderer/render.py

```diff
@@ -111,6 +115,12 @@ def calcGlyphLineBounds(glyphLine, font):
             bounds = glyphBounds
         else:
             bounds = unionRect(bounds, glyphBounds)
+    if useFontMetrics:
+        ttf = font.ttFont
+        x = 0
+        for glyph in glyphLine:
+            x += glyph.xAdvance
+        bounds = (0, ttf["hhea"].descender, x, ttf["hhea"].ascender)
```

Comment:
Please use uharfbuzz 0.20.0 for this as I wrote in my previous review.

Reply:
Sorry not really understanding what you mean or why it matters so much. I want to use the font's metrics, so that means, locked baseline/ascender/descender…to me at least.

Reply:
Variable fonts. Those values may vary, as per MVAR. Going via HB should do this correctly.

Reply:
From the uharfbuzz 0.20.0 release notes:
"Added Font.get_font_extents and FontFuncs.set_font_{v,h}_extents_func methods (https://github.com/harfbuzz/uharfbuzz/issues/106, https://github.com/harfbuzz/uharfbuzz/pull/107)."

Reply:
What should be default direction, and do you want user to pass it in via flag?